src/main/java/net/earthcomputer/multiconnect/impl/APIImpl.java

```diff
@@ -47,26 +51,20 @@ public void removeClientboundStringCustomPayloadListener(ICustomPayloadListener<
         CustomPayloadHandler.removeClientboundStringCustomPayloadListener(listener);
     }
 
-    // TODO: rewrite
-//    @Override
-//    public void forceSendCustomPayload(ClientPlayNetworkHandler networkHandler, Identifier channel, PacketByteBuf data) {
-//        if (networkHandler == null) {
-//            throw new IllegalStateException("Trying to send custom payload when not in-game");
-//        }
-//        CustomPayloadC2SPacket packet = new CustomPayloadC2SPacket(channel, data);
-//        //noinspection ConstantConditions
-//        ((ICustomPayloadC2SPacket) packet).multiconnect_unblock();
-//        networkHandler.sendPacket(packet);
-//    }
-//
+
+    @Override
+    public void forceSendCustomPayload(ClientPlayNetworkHandler networkHandler, Identifier channel, PacketByteBuf data) {
+        if (networkHandler == null) throw new IllegalStateException("Trying to send custom payload when not in-game");
+        CustomPayloadC2SPacket packet = new CustomPayloadC2SPacket(channel, data);
+        ((ICustomPayloadC2SPacket) packet).multiconnect_unblock();
+        networkHandler.sendPacket(packet);
```

Comment:
Although this code may compile, it's going to need to be changed anyway, which is why I marked it for rewrite. Packets aren't going to be blocked this way in the rewrite.